src/gui/addnewtorrentdialog.ui

```diff
@@ -252,35 +252,35 @@
                 </item>
                 <item row="0" column="1">
                  <layout class="QHBoxLayout" name="stopConditionLayout">
-                 <item>
```

Comment:
Those unrelated changes are done by Qt designer.
Here the last person who edited this file didn't indent the code properly.